docs/tutorials/otio-env-variables.md

```diff
@@ -0,0 +1,18 @@
+# Environment Variables
+
+This document describes the environment variables that can be used to configure
+various aspects of OTIO.
+
+## Plugin Configuration
+
+These variables must be set _before_ the OpenTimelineIO python library is imported.
+
+- `OTIO_PLUGIN_MANIFEST_PAT`H: a ":" separated string with paths to .manifest.json files that contain OTIO plugin manifests.  See: <a href="write-an-adapter.html" target="_blank">Tutorial on how to write an adapter plugin</a>.
```

Comment:
typo: _PAT`H

Reply:
Cool.  Using the "insert a suggestion" feature to correct these typos.

```suggestion
- `OTIO_PLUGIN_MANIFEST_PATH`: a ":" separated string with paths to .manifest.json files that contain OTIO plugin manifests.  See: <a href="write-an-adapter.html" target="_blank">Tutorial on how to write an adapter plugin</a>.
```